### PR TITLE
cmd/storagenode: load the identity first during node setup

### LINKED ISSUE #4387: Node in invalid state if identity can't be loaded during setup
When setting up a node with `SETUP="true"`, but with an invalid identity mount location, the node writes out config files, but later fails.

If you run setup again with the correct identity mount, it will exit saying the config already exists.

If you then try and run the node, it will exit with:

```
piecestore monitor: error verifying location and/or readability of storage directory: open config/storage/storage-dir-verification: no such file or directory
```

This could be avoided by moving the identity load (line 304) above the config write.

Move this:

https://github.com/storj/storj/blob/44a73b01d9b0754b037487465d744ebbcf3c976e/cmd/storagenode/main.go#L304

Above line 270.  Then if the during setup the identity is incorrectly mounted it will exit before anything is written and the user can fix it and safely retry the setup command.

### PR DESCRIPTION
**Pull request by onionjake**
This loads the identity before any changes are written to disk. By loading the identity first, then any errors, such as a non-existent identity file, will be discovered before anything is written to disk.  This fixes #4387.

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: cmd/storagenode/main.go [hunk 1]
```diff
@@ -266,6 +266,11 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 	if !valid {
 		return fmt.Errorf("storagenode configuration already exists (%v)", setupDir)
 	}
+
```
Comment (ifraixedes) [inline cmd/storagenode/main.go:269]: CI is telling that this file isn't formatted accordingly gofmt and it's because of this line.

**Comment by ifraixedes** [comment]
@onionjake if you haven't done it yet, please sign the CLA and let us know when you have done it.
Thank youj.

**Comment by profclems** [comment]
Hi @onionjake, thanks so much for working on this! 

Do you mind signing the [CLA](https://docs.google.com/forms/d/e/1FAIpQLSdVzD5W8rx-J_jLaPuG31nbOzS8yhNIIu4yHvzonji6NeZ4ig/viewform) before we can merge this?

**Comment by onionjake** [comment]
I've updated the PR (I did the original in the github web ui, which is why gofmt didn't run).

It might be a few weeks before I can know if I can sign the CLA.  The change is so simple I'm happy to close this out and let someone else propose a patch to fix.

**Comment by ifraixedes** [comment]
Replicated change in commit https://github.com/storj/storj/commit/7f1dc74e370fa624b51c891d1d9a13d0a78bdd79

**Pull request closed without merging**